src/name.cpp

```diff
@@ -146,7 +146,7 @@ std::string generate( bool is_male )
         //~ Used for constructing full name: %1$s is `given name`, %2$s is `family name`
         translation full_name_format = to_translation( "Full Name", "%1$s %2$s" );
         //One in three chance to add a nickname to full name
```

Comment:
May want to update the comment as well.
```suggestion
        //One in ten chance to add a nickname to full name
```

Reply:
done, made the comment more generic, no need to repeat the logic.